torch/_dynamo/variables/torch.py

```diff
@@ -236,15 +236,21 @@ def call_function(
         unspec_python_args = check_unspec_python_args(args, kwargs)
         options = VariableTracker.propagate(self, args, kwargs.values())
 
+        def get_full_name(value):
+            if hasattr(value, "__module__") and hasattr(value, "__name__"):
+                if value.__module__ is not None and value.__name__ is not None:
+                    return value.__module__ + "." + value.__name__
```

Comment:
Seems to cause a lot of breakage. I wonder why.

Reply:
Instead of doing this, let's take the string and manually eval it by doing importlib on the module name and then getattr with the last name. This is similar to how @yanboliang is going to stringify skipfiles/allowlist.

Reply:
I moved the import into dynamo. So dynamo, not the config, does the imports when the dynamo module is being setup.